src/data/thermal-conductivity-constants.ts

```diff
@@ -16,8 +16,404 @@ export const DIPPRThermalConductivityDictionary: Record<
+    minimumTemperature: 257.85,
+    maximumTemperature: 478.6,
+    thermalConductivityAtMinimumTemperature: 0.1615,
+    thermalConductivityAtMaximumTemperature: 0.147,
+  },
+
+  BenzylEthylether: {
+    name: 'Benzyl ethyl ether',
+    formula: 'C9H12O',
+    CAS: '539-30-0',
+    molecularWeight: 136.19098,
+    C1: 0.2029,
+    C2: -0.0002226,
+    minimumTemperature: 275.65,
+    maximumTemperature: 528.6,
+    thermalConductivityAtMinimumTemperature: 0.1415,
+    thermalConductivityAtMaximumTemperature: 0.0852,
+  },
+
+  BenzylMercaptan: {
+    name: 'Benzyl mercaptan',
+    formula: 'C7H8S',
+    CAS: '100-53-8',
+    molecularWeight: 124.20342,
+    C1: 0.20316,
+    C2: -0.00019912,
+    minimumTemperature: 243.95,
+    maximumTemperature: 472.03,
+    thermalConductivityAtMinimumTemperature: 0.1546,
+    thermalConductivityAtMaximumTemperature: 0.1092,
+  },
+
+  Biphenyl: {
+    name: 'Biphenyl',
+    formula: 'C12H10',
+    CAS: '92-52-4',
+    molecularWeight: 154.2078,
+    C1: 0.19053,
+    C2: -0.00015145,
+    minimumTemperature: 342.2,
+    maximumTemperature: 723.15,
+    thermalConductivityAtMinimumTemperature: 0.1387,
+    thermalConductivityAtMaximumTemperature: 0.081,
+  },
+
+  Bromine: {
+    name: 'Bromine',
+    formula: 'Br2',
+    CAS: '7726-95-6',
+    molecularWeight: 159.808,
+    C1: -0.2185,
+    C2: 0.0042143,
+    C3: -0.000017753,
+    C4: 3.1041e-8,
+    C5: -2.0108e-11,
+    minimumTemperature: 266,
+    maximumTemperature: 584,
+    thermalConductivityAtMinimumTemperature: 0.1299,
+    thermalConductivityAtMaximumTemperature: 0.0316,
+  },
+
+  Bromobenzene: {
+    name: 'Bromobenzene',
+    formula: 'C6H5Br',
+    CAS: '108-86-1',
+    molecularWeight: 157.0079,
+    C1: 0.16983,
+    C2: -0.0001981,
+    minimumTemperature: 242.43,
+    maximumTemperature: 429.24,
+    thermalConductivityAtMinimumTemperature: 0.1218,
+    thermalConductivityAtMaximumTemperature: 0.0848,
+  },
+
+  Bromoethane: {
+    name: 'Bromoethane',
+    formula: 'C2H5Br',
+    CAS: '74-96-4',
+    molecularWeight: 108.965,
+    C1: 0.1629,
+    C2: -0.00021198,
+    minimumTemperature: 154.25,
+    maximumTemperature: 327,
+    thermalConductivityAtMinimumTemperature: 0.1302,
+    thermalConductivityAtMaximumTemperature: 0.0936,
+  },
+
+  Bromomethane: {
+    name: 'Bromomethane',
+    formula: 'CH3Br',
+    CAS: '74-83-9',
+    molecularWeight: 94.93852,
+    C1: 0.16143,
+    C2: -0.00021287,
+    minimumTemperature: 179.44,
+    maximumTemperature: 413.15,
+    thermalConductivityAtMinimumTemperature: 0.1232,
+    thermalConductivityAtMaximumTemperature: 0.0735,
+  },
+
+  Butadiene12: {
+    name: '1,2-Butadiene',
+    formula: 'C4H6',
+    CAS: '590-19-2',
+    molecularWeight: 54.09044,
+    C1: 0.21966,
+    C2: -0.0003436,
+    minimumTemperature: 136.95,
+    maximumTemperature: 284,
+    thermalConductivityAtMinimumTemperature: 0.1726,
+    thermalConductivityAtMaximumTemperature: 0.1221,
+  },
+
+  Butadiene13: {
```

Comment:
```suggestion
  '1,3Butadiene`: {
```

See if this is possible.

Reply:
can we Butaiene_1_3

Reply:
sorry , Butadiene_1_3

Reply:
I don't recommend changing the order of the numerics because it may hinder readability. So, try the above suggestion only: `'1,3Butadiene'`.